provider/cmd/pulumi-resource-ec/schema.json

```diff
@@ -1,5 +1,6 @@
 {
     "name": "ec",
+    "displayName": "ElasticCloud (EC)",
```

Comment:
This file is automatically generated. You will want to add the display name field in this struct: https://github.com/pulumi/pulumi-ec/blob/master/provider/resources.go#L79 and then run `make tfgen`